pycode/test/test_epidata_get_divi_data.py

```diff
@@ -153,22 +155,25 @@ class TestGetDiviData(fake_filesystem_unittest.TestCase):
     # data for test dataframe to test the function adjust_data
     d24 = {'bundesland': [1, 2], 'kreis': [1001, 2000], 'ICU': [0, 7]}
     d25 = {'gemeindeschluessel': [1001, 2000], 'ICU': [0, 7]}
-    d262728 = {'bundesland': [1, 2], 'gemeindeschluessel': [1001, 2000], 'ICU': [0, 7]}
-    d29 = {'Unnamed: 0': [0, 0], 'bundesland': [1, 2], 'gemeindeschluessel': [1001, 2000], 'ICU': [0, 7]}
+    d262728 = {'bundesland': [1, 2], 'gemeindeschluessel': [
+        1001, 2000], 'ICU': [0, 7]}
```

Comment:
```suggestion
    d262728 = {'bundesland': [1, 2],
               'gemeindeschluessel': [1001, 2000],
               'ICU': [0, 7]}
```

Reply:
It seems one of you is using the wrong formatter, there are a lot of unnecessary diffs here...